biomodels/README.md

```diff
@@ -0,0 +1,62 @@
+ 
+## Logging
+ 
+This application will log to a file under `./logs/biomodels-{timestamp}.log` and `./logs/biomodels-{timestamp}.err` for warnings and errors. This file will have the same content as the console (stdout/stderr) except without the log4j prefix. 
+ 
+## Configuration
+
+The configuration file is set up like a typical properties file, containing database adaptor information and the database ID of your person instance:
+
+```
+### Sample configuration file with properties
+release.database.host=mySQLHost
+release.database.user=mySQLUsername
+release.database.password=mySQLPassword
+release.database.port=3306
+release_current.name=release_current
+personId=123456
+```
+Once this has been completed and saved somewhere on the server, you should be able to run the biomodels insertion step.
+  
+## Compiling & Running
+
+This is a Java application which requries a Java 8+ environment. You will need maven and a full JDK to compile.
+
+To compile the application, run this command:
+
+```
+$ mvn clean compile package
+```
+
+If this is successful, you should see a JAR file in the `target` directory, with a name like `biomodels-VERSION_NUMBER-jar-with-dependencies.jar`. This is the file you will run.
+
+For example, for version "1.0-SNAPSHOT", execute this command to run the program:
+```
+$ java -jar target/biomodels-1.0-SNAPSHOT-jar-with-dependencies.jar ./config.properties path/to/models2pathways.tsv
+```
+
+If no properties file is specified as the first argument, the program will look in src/main/resources/ for config.properties. Similarly for models2pathways.tsv, unless otherwise specified it will assume it exists in src/main/resources.
+
+## Validating BioModels Insertion </h3>
+
+The insertion step generally takes a few minutes. Once it has completed, the simplest way to verify that it ran correctly is through Reactome's <a href="https://reactome.org/download-data/reactome-curator-tool">Curator Tool</a>. Once you have loaded up the database, first confirm that a 'BioModels Database' instance exists in the `ReferenceDatabase` class. This should have been created during the insertion step, if it didn't exist already. By right-clicking on the BioModels Database instance, you are able to 'Display Referrers'. This should bring up a list of `DatabaseIdentifier` objects that look like 'BioModels Database:BIOMD########'. Confirm that there are no duplicates in this list and compare the count in this release to the previous release. The numbers should be similar, if not a bit higher for the most recent one. That is everything that needs to be done for validation! Next up is the UpdateGKCurrent (link needed) step. 
```

Comment:
It might be good to make the last sentence be its own paragraph so it stands out what the next step is.